usl_models/usl_models/flood_ml/dataset.py

```diff
@@ -0,0 +1,231 @@
+import logging
```

Comment:
Missing module docstring. Could be:
"""tf.data.Datasets for training FloodML model on CityCAT data."""